### DISCUSSION TITLE
Project status

### DISCUSSION THREAD

**Discussion by karolswdev** [LunarVim/LunarVim#4627] [General]
Dear `lvim` team. What's the status of **LunarVim**? I have been enjoying, but grew concerned that the last PR to hit `main` was 6 months ago. I also notice that there hasn't been any movement on the Project page. Is the project dead? Is there a roadmap?

**Comment by JosephBrooksbank** [comment]
https://github.com/LunarVim/LunarVim/discussions/4518#discussioncomment-8963843
